### ISSUE TITLE
Add ability to use key in Hallway

### ISSUE THREAD

**Issue by Tesfa-Worku** [luceroweb/haunted-house-game#48]
[timebox: 30, complexity: 1]
As a player, I want to have the option to present the key in the hallway when I have a key so that I win.

**Assets**
- Gold key boolean: ‘hasGoldKey’ (default false)
- Silver key boolean: ‘hasSilverKey’ (default false)
- Hallway component: Hallway.js
- KeyDisplay function: KeyDisplay.js
- ShowKey function: ShowKey.js

**Expected outcome**
-  If the player has the gold key or the silver key, a button in Hallway using ShowKey (function that shows story message) appears
- KeyDisplay has message saying to return to hallway and open the door

**Comment by luceroweb** [comment]
@Tesfa-Worku how is this ticket different from #9? Please update the ticket's description to clarify.

**Comment by Tesfa-Worku** [comment]
@luceroweb I don't think we had a separate page for win or (gold key case) on last week's assignment. If we have them, I have to close this ticket. My assigned project involves displaying a false win page too.

**Comment by xmercado** [comment]
@luceroweb me, @Tesfa-Worku , and @garanico have discussed and edited the ticket for more precision and clarity.
